src/Amazon.AspNetCore.DataProtection.SSM/PersistOptions.cs

```diff
@@ -28,5 +28,8 @@ public class PersistOptions
         /// don't specify a key ID, the system uses the default key associated with your AWS account.
         /// </summary>
         public string KMSKeyId { get; set; }
+
+        /// <summary>The prefix applied to the DataProtection key names.</summary>
+        public string ParameterNamePrefix { get; set; }
```

Comment:
That said, I'm not a fan of how this is hanging around like a trap on the paths for which `parameterNamePrefix` is provided as a direct argument.

I'm not sure what the taste of the team for a 2.0 release is, so I've left it this way for now. If 2.0 is desirable (and for using ASP.NET Core idioms, I think it could be justified), please do let me know. There is more I'd like to be able to do.